packages/debugger/lib/web3/actions/index.js

```diff
@@ -1,8 +1,16 @@
 export const INIT_WEB3 = "INIT_WEB3";
-export function init(provider) {
+export function init(provider, ensRegistryAddress) {
```

Comment:
I see there's a bunch of these `ensRegistryAddress`. You might be a bit concerned by my suggesting that we'll have other `ens` settings in the future... happy to get on a call about it, but I'll just suggest that it's pretty common these days for applications to display mainnet ENS names even when connected to other networks. We'll undoubtedly want to follow best practices here.

Reply:
I mean I guess for consistency we can also do that here?